### PR TITLE
Add some notes to the Sphinx 6 post

### PR DESCRIPTION
**Pull request by benjaoming**

### CODE REVIEW

**Review by ericholscher — COMMENTED** [review]
This seems a bit too "point in time", and we will not remember to update it again. I think we just want to update it to mention the Sphinx 6.1 release, and our theme final release when it happens?

File: sphinx6-upgrade.rst [hunk 1]
```diff
@@ -11,6 +11,19 @@
 Sphinx 6 is out and has important breaking changes
 ==================================================
 
+.. note::
+
+   **Important updates to this post ⬇️**
+   
+   - `sphinx-rtd-theme`_ 1.2.0rc4 has been released (to be followed by a final 1.2.0 release).
```
Comment (benjaoming) [inline sphinx6-upgrade.rst:18]:
```suggestion
   - `sphinx-rtd-theme`_ 1.2.0 has been released.
```

File: sphinx6-upgrade.rst [hunk 2]
```diff
@@ -11,6 +11,19 @@
 Sphinx 6 is out and has important breaking changes
 ==================================================
 
+.. note::
+
+   **Important updates to this post ⬇️**
+   
+   - `sphinx-rtd-theme`_ 1.2.0 has been released.
```
Comment (ericholscher) [inline sphinx6-upgrade.rst:18]:
```suggestion
   - `sphinx-rtd-theme`_ 1.2.0 has been released with support for Sphinx 6.
```

**Review by ericholscher — APPROVED** [review]

**Comment by benjaoming** [comment]
Merging - we should keep a close eye on our failing list, I'll set a reminder :)

**Pull request merged**